binstar_client/commands/login.py

```diff
@@ -118,4 +153,6 @@ def add_parser(subparsers):
                            help="Specify your password. "
                                 "If this is not given, you will be prompted"
                            )
+    subparser.add_argument('--kerberos', action='store_true',
```

Comment:
This should probably be a config variable. 

Also, could the client this with the server? 

e.g. a pre step: `[GET] api/auth/authentication-type`